docs/src/examples/components/Header/Variations/HeaderExampleDescriptionCustomization.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { Header } from '@stardust-ui/react'
 
-const HeaderExampleSubheader = () => (
+const HeaderExampleDescriptionCustomization = () => (
```

Comment:
Note, all example names need to match file names, which should match the title of the example.  We could use a conformance test around this.